components/ingest-service/backend/elastic/mappings/node_state.go

```diff
@@ -81,6 +81,9 @@ var nodeProps = `
 	"status": {
 		"type": "keyword"
 	},
+	"chef_run_status": {
+		"type": "keyword"
+	},
```

Comment:
In case anyone wonders, I'm expecting this to be an additive change to the node state, no migration. There could be nodes that were ingested prior to upgrading to a version of code with the `chef_run_status` that are/become missing, but these won't be relevant in the change tracking APIs anyway.